packages/recommend-react/src/useFrequentlyBoughtTogether.ts

```diff
@@ -23,9 +23,13 @@ export function useFrequentlyBoughtTogether<TObject>({
   });
   const { status, setStatus } = useStatus('loading');
   const objectIDs = useStableValue(userObjectIDs);
-  const transformItems = useStableValue(userTransformItems);
   const queryParameters = useStableValue(userQueryParameters);
 
+  const transformItemsRef = useRef(userTransformItems);
+  useEffect(() => {
+    transformItemsRef.current = userTransformItems;
+  }, [userTransformItems]);
```

Comment:
So that the `useAlgoliaAgent()` call doesn't get missed/lost, perhaps we can move this code below it.

Reply:
✅ : https://github.com/algolia/recommend/pull/110/commits/9214aa08f43745d5dda1736e1fc1dd4118f5e038